resources/js/components/modals/AdminShowConversation.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div>
+        <Loading v-if="loading" />
+        <template v-else>
+            <AdminShowMessage v-for="message in conversation.messages" :key="message.id"
+                              :message="message" />
+        </template>
+    </div>
+</template>
+
+<script>
+import {mapGetters} from 'vuex';
+import AdminShowMessage from '../small/AdminShowMessage.vue';
+import Loading from '../../components/Loading.vue'
+export default {
+    components: {
+        AdminShowMessage, Loading,
+    },
+    props: {
+        conversationId: {
+            type: Number,
```

Comment:
It's sent as a string. It works either way, but it does give a console error, so change this to String

Reply:
changed the resource so it sends as an int

Reply:
shouldn't this be changed to an int in the data base though?

Reply:
Initially I had intended for a random string of letters and numbers, which is why it's a string.